src/content/docs/en/pages/guides/marketplace/templates/devscard.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: How to deploy DevsCard using a template
+description: >-
+  This template contains the configurations to create your online and print resume using the DevsCard project.
+meta_tags: >-
+  templates, guides, Azion Marketplace, resume, portfolio, DevsCard, web, print
+namespace: docs_guides_devscard
+permalink: /documentation/products/guides/devscard/
```

Comment:
```suggestion
permalink: /documentation/products/guides/devscard-template/
```

Reply:
to be more specific and avoid duplication in case of new cases/features related to DevsCard